gcovr/decision_analysis.py

```diff
@@ -156,11 +156,12 @@ def parse_one_line(self, lineno: int, code: str) -> None:
         if len(line_coverage.branches.items()) > 0:
             if _is_a_loop(code) or _is_a_oneline_branch(code):
                 if len(line_coverage.branches.items()) == 2:
+                    keys = list(line_coverage.branches.keys())
```

Comment:
In practice this is correct, but using `sorted()` might make it clearer that we want the two smallest keys:

```suggestion
                    keys = sorted(line_coverage.branches)
```

Reply:
We actually want only two, not particularly the smallest, because the previous condition already guarantees only two branches. But if ```sorted()``` improves readability, I will change this.

Reply:
From Python 3.7 on the result is the same. In previous versions the inserted order isn't preserved and the branches can be in a different order. With `sorted` they are always ordered by the number as in the gcov output.